--- glm/CMakeLists.txt.orig	2018-10-13 08:46:37.872559133 +0200
+++ glm/CMakeLists.txt	2018-10-13 09:11:42.575875287 +0200
@@ -236,13 +236,13 @@
 # install tree package config
 configure_package_config_file(
 	cmake/glmConfig.cmake.in
-	${GLM_INSTALL_CONFIGDIR}/glmConfig.cmake
+	${CMAKE_CURRENT_BINARY_DIR}/glmConfig.cmake
 	INSTALL_DESTINATION ${GLM_INSTALL_CONFIGDIR}
 	PATH_VARS CMAKE_INSTALL_INCLUDEDIR
 	NO_CHECK_REQUIRED_COMPONENTS_MACRO)
 
 install(FILES
-	"${CMAKE_CURRENT_BINARY_DIR}/${GLM_INSTALL_CONFIGDIR}/glmConfig.cmake"
+	"${CMAKE_CURRENT_BINARY_DIR}/glmConfig.cmake"
 	"${CMAKE_CURRENT_BINARY_DIR}/glmConfigVersion.cmake"
 	DESTINATION ${GLM_INSTALL_CONFIGDIR})
 
