--- glm-1.0.1/test/core/core_func_matrix.cpp.orig	2024-02-27 18:19:47.000000000 +0100
+++ glm-1.0.1/test/core/core_func_matrix.cpp	2024-05-30 13:20:49.089552367 +0200
@@ -392,7 +392,7 @@ static int test_inverse_perf(std::size_t
 	//glm::uint Ulp = 0;
 	//Ulp = glm::max(glm::float_distance(*Dst, *Src), Ulp);
 
-	std::printf("inverse<%s>(%f): %lu\n", Message, static_cast<double>(Diff), EndTime - StartTime);
+	std::printf("inverse<%s>(%f): %llu\n", Message, static_cast<double>(Diff), (unsigned long long)(EndTime - StartTime));
 
 	return 0;
 }
